src/pages/docs/projects/steps/conditions/index.mdx

```diff
@@ -161,6 +161,8 @@ This functionality is available on all other steps.
 ### Retries
 Enabling Retries gives you the ability to automatically retry a step if it fails, with up to three attempts. This feature is particularly useful when dealing with steps that commonly fail due to temporary or transient errors during deployment.
 
+When a step fails the server will wait for a "backoff interval" to pass before trying again. By default, the backoff interval is 15 seconds, however this value can be adjusted as required.
```

Comment:
It would be great if you could add "can be adjusted [...how]" here.

Reply:
Thanks Steve. I've improved the text and added an updated screenshot